src/python/pants/option/option_types.py

```diff
@@ -556,7 +546,6 @@ def __new__(
             deprecation_start_version=deprecation_start_version,
             daemon=daemon,
             fingerprint=fingerprint,
-            environment_sensitive=environment_sensitive,
```

Comment:
Do you need to update I think `parser.py` to remove this as a valid kwarg?

Reply:
Yup, good call